--- ../db-6.2.32/lang/php_db4/db4.cpp	2017-04-13 23:06:15.000000000 +0900
+++ db4.cpp	2018-01-13 20:09:35.015911015 +0900
@@ -1681,7 +1681,7 @@ ZEND_NAMED_FUNCTION(_wrap_dbc_get)
     DBC *dbc;
     DBT key, value;
     zval *zkey, *zvalue;
-    u_int32_t flags = DB_NEXT;
+    long flags = DB_NEXT;
     zval *self;
     int ret;
     
@@ -1905,9 +1905,9 @@ ZEND_NAMED_FUNCTION(_wrap_db_env_open)
     zval *self;
     char *home = NULL;
     long  homelen;
-    u_int32_t flags = DB_CREATE  | DB_INIT_LOCK | DB_INIT_LOG | \
+    long flags = DB_CREATE  | DB_INIT_LOCK | DB_INIT_LOG | \
             DB_INIT_MPOOL | DB_INIT_TXN ;
-    int mode = 0666;
+    long mode = 0666;
     int ret;
 
     getDbEnvFromThis(dbenv);
